libraries/rush-lib/src/cli/scriptActions/PhasedScriptAction.ts

```diff
@@ -303,6 +310,12 @@ export class PhasedScriptAction extends BaseScriptAction<IPhasedCommandConfig> {
         'This directory will contain a subdirectory for each project and phase.'
     });
 
+    this._debugBuildCacheIdsParameter = this.defineFlagParameter({
```

Comment:
Only include if build cache is enabled? Or do we know that yet?

Reply:
We don't know that yet, so I can't make it conditionally defined.